File: app/src/main/java/com/example/android/radiancex/AccountFragment.java
package com.example.android.radiancex;

import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;

import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentTransaction;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;

public class AccountFragment extends Fragment {

    Button btnGoToAccountSettings;

    public AccountFragment() {
        // Required empty public constructor
    }


    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        View view = inflater.inflate(R.layout.fragment_account, container, false);
        btnGoToAccountSettings = view.findViewById(R.id.goToAccountSettings);
        btnGoToAccountSettings.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse("https://github.com/vuhoangthaiduong"));
                startActivity(intent);
            }
        });
        return view;
    }
}

File: app/src/main/java/com/example/android/radiancex/AccountFragment.kt
package com.example.android.radiancex

import android.content.Intent
import android.net.Uri
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.Button
import androidx.fragment.app.Fragment

class AccountFragment : Fragment() {
    var btnGoToAccountSettings: Button? = null

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        // Inflate the layout for this fragment
        val view = inflater.inflate(R.layout.fragment_account, container, false)
        btnGoToAccountSettings = view.findViewById(R.id.goToAccountSettings)
        btnGoToAccountSettings.setOnClickListener(View.OnClickListener {
            val intent = Intent(Intent.ACTION_VIEW, Uri.parse("https://github.com/vuhoangthaiduong"))
            startActivity(intent)
        })
        return view
    }
}